src/universal_transfer_operator/data_providers/database/snowflake.py

```diff
@@ -15,19 +15,28 @@
 from universal_transfer_operator.universal_transfer_operator import TransferIntegrationOptions
 
 
+@attr.define
+class SnowflakeOptions(TransferIntegrationOptions):
+    """
+    Snowflake load option for naive transfer.
```

Comment:
Can you add the documentation URL for snowflake options?

Reply:
Will in native support PR - https://github.com/astronomer/apache-airflow-provider-transfers/pull/42